core/state_transition.go

```diff
@@ -363,6 +370,27 @@ func (st *StateTransition) preCheck() error {
 			}
 		}
 	}
+
+	// Check that auth list isn't empty.
+	if msg.AuthList != nil && len(msg.AuthList) == 0 {
+		return fmt.Errorf("%w: address %v", ErrEmptyAuthList, msg.From.Hex())
+	}
+
+	// TODO: remove after this spec change is merged:
```

Comment:
https://github.com/ethereum/EIPs/pull/8845 merged
so we can delete the following code?
I think the check for [y_parity, r, s] have been done when `auth.Authority()` is called.